contrib/download-frozen-image-v2.sh

```diff
@@ -63,10 +63,10 @@ fetch_blob() {
 			-o "$targetFile" \
 			-D-
 	)"
-	curlHeaders="$(echo "$curlHeaders" | tr -d '\r')"
-	if [ "$(echo "$curlHeaders" | awk 'NR == 1 { print $2; exit }')" != '200' ]; then
+	if echo "$curlHeaders" | egrep -q "^HTTP/[0-9].[0-9] 307"; then
```

Comment:
```
:~$ cat `which egrep`
#!/bin/sh
exec grep -E "$@"
```

We can still use these variants, since they provided for backward compatibility.